### ISSUE TITLE
SCION Wireshark pluging should not use ProtoExpert.new with empty second arg

### ISSUE THREAD

**Issue by klausman** [scionproto/scion#2000]
Error message as of Wireshark 2.6.x:

```
 [string "/usr/lib/x86_64-linux-gnu/wireshark/plugins/2.6/scion.lua"]:194: bad argument #2 to 'new' (Empty text)
```

I don't know what the text is used for, but apparently it's not allowed to be empty

**Comment by shitz** [comment]
@klausman can you please verify that this has been fixed by #1925 for you?

**Comment by klausman** [comment]
Fixed for me.
